packages/compass-aggregations/src/plugin.jsx

```diff
@@ -29,15 +28,25 @@ class Plugin extends Component {
       <ConfirmationModalArea>
         <Provider store={this.props.store}>
           <Aggregations
-            showExportButton={this.props.showExportButton}
-            showRunButton={this.props.showRunButton}
-            showExplainButton={this.props.showExplainButton}
+            showExportButton={this.props.enableImportExport}
+            showRunButton={this.props.enableAggregationBuilderRunPipeline}
+            showExplainButton={this.props.enableExplainPlan}
           />
         </Provider>
       </ConfirmationModalArea>
     );
   }
 }
 
+const Plugin = withPreferences(
+  AggregationsPlugin,
+  [
+    'enableImportExport',
+    'enableAggregationBuilderRunPipeline',
+    'enableExplainPlan',
```

Comment:
Before this change, compass collection store would pass those to the view when setting up roles which relied on knowing exact name of the tab, I could just keep this logic, but it felt kinda weird to do it, putting this in preferences makes more sense to me as those are features that are disabled in DE environment and it feels like it's a good idea to have single source of truth for enabling them between query bar and aggregations plugin.

This is not breaking for DE, their default coming from compass-preferences right now should always be `undefined`, and when we do the compass web integration the plan is that we will provide a default config so that preferences don't have unexpected behavior